platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java

```diff
@@ -116,9 +116,9 @@
 import org.apache.logging.log4j.Logger;
 
 /**
- * The Browser that launches the Platforms that run the apps. The Browser has only one public method, which
- * normally does nothing. See the javadoc on the main method for how it can be useful for an app to call it
- * during app development.
+ * The Browser that launches the Platforms that run the apps. The Browser has only one public method, which normally
```

Comment:
This comment is no longer accurate (multiple methods are public)

Reply:
Deleted outdated comments.